python/fledge/services/core/server.py

```diff
@@ -1246,6 +1282,18 @@ async def get_auth_token(cls, request: web.Request) -> web.Response:
             :Example:
                 curl -sX GET http://localhost:<core mgt port>/fledge/service/authtoken?_vid=<>
         """
+        bearer_token = request.headers.get('bearer_token', "")
+        if bearer_token:
+            services_list = ServiceRegistry.all()
+            is_bearer_token_valid = False
+            for svc in services_list:
+                if svc._token == bearer_token:
```

Comment:
this check should be fixed

Reply:
This is not related to the code changed so far.

If bearer_token has in "sub" claim the service name then ServiceRegistry.getBearerToken(claims['sub'])
can be used.

Could you check this?

Reply:
Yes, And can be a common code as we see in other methods.